L10nSnapshotTests/L10nSuite2SnapshotTests.swift

```diff
@@ -112,23 +112,31 @@ class L10nSuite2SnapshotTests: L10nBaseSnapshotTests {
         waitForExistence(app.cells["Logins"], timeout: 15)
         app.cells["Logins"].tap()
 
-        waitForExistence(app.navigationBars.buttons.firstMatch, timeout: 15)
+        // First time only: The message "Your passwords are now protected
+        // by Face ID..." is present when the Firefox app is run after the
+        // simulator has been erased.
+        waitForExistence(app.navigationBars.element(boundBy: 0), timeout: 3)
+        waitForExistence(app.otherElements.buttons.element(boundBy: 2))
         app.otherElements.buttons.element(boundBy: 2).tap()
 
         let passcodeInput = springboard.secureTextFields.firstMatch
         waitForExistence(passcodeInput, timeout: 20)
         passcodeInput.tap()
+        sleep(1)
         passcodeInput.typeText("foo\n")
 
         waitForExistence(app.tables["Login List"], timeout: 10)
         app.buttons.element(boundBy: 1).tap()
         waitForExistence(app.tables["Add Credential"], timeout: 10)
         snapshot("CreateLogin")
         app.tables["Add Credential"].cells.element(boundBy: 0).tap()
+        sleep(1)
         app.keyboards.keys.element(boundBy: 3).tap()
         app.tables["Add Credential"].cells.element(boundBy: 1).tap()
+        sleep(1)
```

Comment:
Have you tried a waitForExistence of the keys instead of sleep()?
I will add that too before line 141, `app.navigationBars["Client.AddCredentialView"]`

Reply:
Yes, I've tried `waitForExistence(app.keyboards.keys.element(boundBy: 0))`. The existence of the key does not mean that it is hittable. 🤷🏼‍♀️